tests/Application/ConfigurationTest.php

```diff
@@ -36,6 +36,8 @@ public static function setUpBeforeClass(): void
         self::initializeApplication(['test_case' => 'TestApplication']);
 
         self::$configuration = self::getContainer()->get('sofascore.purgatory.configuration_loader')->load();
+
+        self::ensureKernelShutdown();
```

Comment:
We need to unregister the exception handler before [PHPUnit creates a snapshot](https://github.com/sebastianbergmann/phpunit/blob/4c40dd02c6146fe9ed0b540f25724ce82d0baabd/src/Framework/TestCase.php#L1897-L1900).